src/components/AppControls.vue

```diff
@@ -344,8 +344,19 @@ export default {
 
 <style scoped>
 .wrapper {
+    /* Sticky is better than fixed because fixed takes the element out of flow,
+     which breaks the height, putting elements underneath */
+    position: sticky;
+
+    /* This is competing with the recipe instructions which have z-index: 1 */
+    z-index: 2;
+
+    /* The height of the nextcloud header */
+    top: 50px;
```

Comment:
I honestly do not like to do it this way. Once, the styling of the server gets changed, we will run into trouble. But for now, let's keep it simple and hope for the best.

Reply:
I agree. I dislike such magic numbers. I looked for some API that nextcloud core would expose, but I didn't find anything.

Edit: I just looked through the `:root` variables in dev tools and found `var(--header-height)` which is set to `50px`.

Reply:
This is resolved in #897.